client/my-sites/pages/page-list.jsx

```diff
@@ -202,7 +213,7 @@ var Pages = React.createClass( {
 			rows = [];
 
 		// pages have loaded, sites have loaded, and we have a site instance or are viewing all-sites
-		if ( pages.length && this.props.sites.initialized ) {
+		if ( pages.length && this.props.hasSites ) {
```

Comment:
> I'm seeing the results for "all sites" until that SELECTED_SITE_SET action gets dispatched.

@jblz, I'm not able to reproduce this, unless I misinterpreted:

![pages-unsiteslist-2](https://cloud.githubusercontent.com/assets/150562/25743555/de506308-318c-11e7-94e1-026c82fe1e15.gif)

(if you like, we can sync up once you're around)